docs/tutorials/basic-dapp/part-4-use-external-accounts.md

```diff
@@ -0,0 +1,189 @@
+
+## 4.1 Get Polkadot-js extension
+
+Get into our user's shoes and install the [Polkadot-js enxtension](https://github.com/polkadot-js/extension) from the [Chrome](https://chrome.google.com/webstore/detail/polkadot%7Bjs%7D-extension/mopnmbcafieddcagagdcbnhejhlodfdd?hl=en) or [Firefox](https://addons.mozilla.org/en-US/firefox/addon/polkadot-js-extension/) store. 
+Create an account using the extension and give it a name.
+
+We will now adapt our application to inject any account created externally in our current interface.
+
+This is how the extension looks like with 2 accounts creatively named Bob and Alice:
+![Pokadot-js extension](/docs/tutorials/basic-dapp/assets/part-3-1.jpg)
+
+## 4.2 Display external account's balances
+
+Externally generated accounts loaded thank to [`@polkadot/extension-dapp`](https://github.com/polkadot-js/extension/tree/master/packages/extension-dapp).
+```bash
+yarn add @polkadot/extension-dapp@beta
+```
+
+This package gives us access to `web3Enable` to declare our DApp and request the right to read the accounts from our users. `isWeb3Injected` lets us know if our users have an extension supporting the `@polkadot/extension-dapp` protocol, and finally the `web3Accounts` function that should be called if the user has a compatible extension, and has granted us access to their accounts.
+
+To keep things simple, we will not display anything else than a loader to our users while the authorization is requested. This is a blocking experience that should not be reproduced in a real DApp. Rather show something to your users, and patiently wait for them to accept the extension request.
+
+We will introduce a new state variable `accountLoaded` to show this loader in case our user have an extension but haven't granted us access to their accounts.
+
+```js
+// Toward the top of Apps.js
+// new state variables to monitor the account loading status
+const [accountLoaded, setaccountLoaded] = useState(false);
+
+// in our render function, show a loader to users with a compatible extension while 
+// waiting for their authorization
+if (!accountLoaded) {
+return loader('Loading accounts (please review any extension\'s authorization)');
+}
+```
+
+We will declare our DApp and request access to the external accounts in a `useEffect` React hook that we are now familiar with. We used to call `loadAccounts()` to initialize the keyring. This function also accepts injected accounts as arguments, so all we'll have to do is `loadAccounts(injectedAccounts)`. Quite handy isn't it?
```

Comment:
Starting a war on commas here. I'd remove all of the ones before `"and"` and add one here for example `"Quite handy, isn't it"`, but I'm not really familiar with the english comma rules. That would just be me adapting the german rules to english